packages/fury-adapter-swagger/lib/parser.js

```diff
@@ -366,6 +366,45 @@ class Parser {
           });
         }
 
+        if (this.swagger.info.contact) {
+          this.withPath('contact', () => {
+            const { Link } = this.namespace.elements;
+            const { name, url, email } = this.swagger.info.contact;
+
+            if (url) {
```

Comment:
I spotted below that the source maps for the URL and the email are the same, we can fix this by wrapping the URL and the email blocks in `this.withPath('url', () => {` and `this.withPath('email' => {` respectively.

```suggestion
            if (url) {
              this..withPath('url', () => {
                  ...
              });
            }
```

Afterwards, the source maps in the test fixture will show for the url and the email differently, as right now `this.path` is just set to `info.contact` so the source map is for the contact object not the specific value.